### DISCUSSION TITLE
Data not showing in Mysql even if the data exist in table

### DISCUSSION THREAD

**Discussion by cherrycharan** [Budibase/budibase#6587] [Help]
I have connected MYSQL DB and when i open the table to view the data, it shows that there is no data but data already exist in MYSQL DB table
<img width="962" alt="175515526-8a4f724e-18d5-400c-848e-57611c3512e6" src="https://user-images.githubusercontent.com/16544980/177634422-0c656ecb-055b-46b7-bdab-d57ba8900f5b.png">

**Comment by mike12345567** [comment]
Hi @cherrycharan - if you can provide the DDL for one of your tables which doesn't work, for example
```sql
CREATE TABLE employees (
    id            INTEGER       PRIMARY KEY,
    first_name    VARCHAR(50)   not null,
    last_name     VARCHAR(75)   not null,
    mid_name      VARCHAR(50)   not null,
    dateofbirth   DATE          not null
);
```
Something in this format, then then we can try replicate and see what might be causing the problem.
